ReleaseNotes.md

```diff
@@ -1,224 +1,348 @@
+* [Open Sans Light font](http://www.google.com/fonts/specimen/Open+Sans)
+
 ### MIT License ###
+
+* [Buzz v1.1.0](http://buzz.jaysalvat.com)
 * [Foundation v6.2.3](http://foundation.zurb.com/index.html)
 * [HTML5 Boilerplate v4.3](http://html5boilerplate.com/)
+* [html5shiv v3.7.2](https://code.google.com/p/html5shiv/)
 * [jQuery v3.1.0](http://jquery.com/)
-* [jQuery UI (Core; Interactions: draggable, resizable; Widgets: button, checkboxradio, controlgroup, dialog, mouse, slider, tabs, and tooltip) v1.12.1](http://ui.jquery.com/)
+* [jQuery Mockjax v2.2.1](https://github.com/jakerella/jquery-mockjax)
 * [jQuery QUnit v1.12.0](http://qunitjs.com)
 * [jQuery QUnit Composite v1.0.1](https://github.com/jquery/qunit-composite)
-* [jQuery Mockjax v2.2.1](https://github.com/jakerella/jquery-mockjax)
 * [jQuery scrollTo v1.4.2](http://flesler.blogspot.com/2007/10/jqueryscrollto.html)
 * [jQuery Touch Punch v0.2.2](http://touchpunch.furf.com/)
-* [jquery.simulate](https://github.com/eduardolundgren/jquery-simulate)
+* [jQuery UI (Core; Interactions: draggable, resizable; Widgets: button, checkboxradio, controlgroup, dialog, mouse, slider, tabs, and tooltip) v1.12.1](http://ui.jquery.com/)
 * [jquery.selectbox v0.5 (forked)](https://github.com/fluid-project/jquery.selectbox)
+* [jquery.simulate](https://github.com/eduardolundgren/jquery-simulate)
 * [Micro Clearfix](http://nicolasgallagher.com/micro-clearfix-hack/)
 * [Normalize v4.1.1](https://necolas.github.io/normalize.css/)
-* [Buzz v1.1.0](http://buzz.jaysalvat.com)
-* [html5shiv v3.7.2](https://code.google.com/p/html5shiv/)
-
-### zlib/libpng License ###
-* [fastXmlPull is based on XML for Script's Fast Pull Parser v3.1](http://wiki.fluidproject.org/display/fluid/Licensing+for+fastXmlPull.js)
-
-### ECL 2.0 ###
-* Sample markup and stylesheets from [Sakai v2.5](http://sakaiproject.org)
-
-### Apache 2.0 ###
-* [Open Sans Light font](http://www.google.com/fonts/specimen/Open+Sans)
 
 ### Public Domain ###
+
 * Douglas Crockford's [JSON.js (from 2007-11-06)](http://www.json.org/)
 
-## Documentation ##
+### zlib/libpng License ###
 
-We are in the process of migrating our documentation to a new space. The markdown files for the documentation can be found in [github](https://github.com/fluid-project/infusion-docs).
+* [fastXmlPull is based on XML for Script's Fast Pull Parser v3.1](http://wiki.fluidproject.org/display/fluid/Licensing+for+fastXmlPull.js)
 
-The new space is dedicated to only Infusion documentation, and provides improved navigation.
+## Documentation ##
 
-Some of our documentation remains in the wiki space: Links to these pages are indicated with _**wiki**_. From any of these pages, you can return to the main documentation space using your
-browser's Back button.
+Documentation and tutorials can found on the [Infusion Documentation](http://docs.fluidproject.org/infusion/development/) site.
```

Comment:
If we haven't already, can we make sure that all of the old documentation pages in our wiki have had their content removed and prominent redirects or notices placed on them to the docs site?